test/utils.cpp

```diff
@@ -363,18 +363,6 @@ TEST_CASE("get_command_output()", "[utils]")
 	REQUIRE(utils::get_command_output("echo c\" d e") == "");
 }
 
-TEST_CASE("extract_filter()", "[utils]")
```

Comment:
Please don't remove this test: we need it to make sure that the C++ bindings work okay.

Reply:
Updated with test recovered.